src/script/components/app-header.ts

```diff
@@ -42,6 +41,13 @@ export class AppHeader extends LitElement {
           color: black;
         }
       }
+
+      @media(max-width: 542px) {
```

Comment:
Nit: I dislike having hard-coded values like this sprinkled across our files. Is there something magical about 542? If so, should it be in a CSS variable in our global CSS file?

Reply:
Yep, so this is what I was talking about above: You will notice that I added and then removed some CSS vars for breakpoints, I need to investigate why that was not working further. Also, the breakpoints I am following are here https://developer.microsoft.com/en-us/fluentui#/styles/web/layout with the mobile one slightly tweaked to catch the Surface Duo.

Reply:
Basically, when I had them as CSS vars they werent working with media queries, doing research as we speak.

Reply:
Looking at [this issue](https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries), they say variables won't work for media queries. 😢

They suggest using a CSS preprocessor like Less or Sass.